project/templates/app/mobi/app/includes/comments_include.html

```diff
@@ -10,7 +10,7 @@
       {% for comment in comments %}
           <div class="item comment no-pic">
               <div class="meta">
-                  <div class="title">{{ comment.name }}</div>
+                  <div class="title">{% if comment.user.profile.engage_anonymously %}{{ comment.name }} {% else %} {{comment.user.username}}{%endif%}</div>
```

Comment:
Please perform this check in the [views.py](https://github.com/praekelt/actionforwomen/blob/develop/app/views.py#L760-L765) instead of the templates so that the anonymous settings doesn't divulge historical comments.